src/hyperion/parameters/plan_specific/gridscan_internal_params.py

```diff
@@ -59,6 +60,12 @@ def _preprocess_experiment_params(
         cls,
         experiment_params: dict[str, Any],
     ):
+
+        #Force first grid scan to have even number of rows so next grid starts at same point
```

Comment:
Could: I think maybe a bit more explanation here as to why we need to do it would be good.